packages/react/src/global-nav-item/GlobalNavItem.tsx

```diff
@@ -0,0 +1,61 @@
+import * as RadixCollapsible from "@radix-ui/react-collapsible";
+import { forwardRef, useContext } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import { GlobalNavContext } from "../global-nav-context";
+import { extractSprinkles } from "../sprinkles";
+
+export type GlobalNavItemProps = BoxProps<
+  typeof Button,
+  {
+    active?: boolean;
+  }
+>;
+
+export const GlobalNavItem = forwardRef<HTMLButtonElement, GlobalNavItemProps>(
+  ({ active, children, endDecorator, startDecorator, ...props }, ref) => {
+    const { restProps, sprinkleProps } = extractSprinkles(props);
+
+    const { expanded } = useContext(GlobalNavContext);
+
+    return (
+      <Flex
+        alignItems="center"
+        asChild
+        flexDirection="row"
+        gap="sm"
+        h="lg"
+        px="sm"
+        py="xs"
+        w="full"
+        {...sprinkleProps}
+      >
+        <Button
+          appearance="secondary"
```

Comment:
We might need to add a gap here as there's a gap of `12` in figma between icon and text.